### PR TITLE
f-searchbox@4.0.0-beta.27 - Replace knobs with controls and allow changing of locale in storybook

### PR DESCRIPTION
**Pull request by charliearlie**
### Added
- Locale switching to `searchbox` inside Storybook

## Browsers Tested

- [x] Chrome (latest)
- [x] Mobile (Please list device/browser – Ideally one iPhone model and one Android)

### List any other browsers that this PR has been tested in:

- [View the Browser Support Checklist](http://fozzie.just-eat.com/documentation/general/browser-support)

### CODE REVIEW

**Review by kevinrodrigues — DISMISSED** [review]

**Review by simonsandersje — DISMISSED** [review]

File: packages/components/molecules/f-searchbox/stories/Searchbox.stories.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-    select, boolean
-} from '@storybook/addon-knobs';
 import Vue from 'vue';
 import Vuex from 'vuex';
 import { withA11y } from '@storybook/addon-a11y';
+import {
+    ENGLISH_LOCALE,
+    SPANISH_LOCALE
+} from '@justeat/storybook/constants/globalisation';
+
```
Comment (AlixRichards) [inline packages/components/molecules/f-searchbox/stories/Searchbox.stories.js:5]: Is it possible to add the globalisation for the other tenants too? So IE, IT, AU, NZ, DK & NO as well

**Review by AlixRichards — COMMENTED** [review]
Looks good! Just added a comment about the other tennants

File: packages/components/molecules/f-searchbox/stories/Searchbox.stories.js (hunk 1 above)
Comment (charliearlie) [reply packages/components/molecules/f-searchbox/stories/Searchbox.stories.js:5]: Of course. I just used the ones there were constants for, so wasn't sure what else there was. But I've found a small list in the `cookie-banner` stories.

Comment (AlixRichards) [reply packages/components/molecules/f-searchbox/stories/Searchbox.stories.js:5]: Amazing, thank you!

File: packages/storybook/constants/globalisation.js [hunk 2]
```diff
@@ -1,4 +1,10 @@
+export const AUSTRALIAN_LOCALE = 'en-AU';
+export const DANISH_LOCALE = 'da-DK';
 export const ENGLISH_LOCALE = 'en-GB';
+export const IRISH_LOCALE = 'en-IE';
+export const ITALIAN_LOCALE = 'it-IT';
+export const NEW_ZEALAND_LOCALE = 'en-NZ';
+export const NORWEIGAN_LOCALE = 'nb-NO';
 export const SPANISH_LOCALE = 'es-ES';
 
 export const VALID_LOCALES = [ENGLISH_LOCALE, SPANISH_LOCALE];
```
Comment (charliearlie) [inline packages/storybook/constants/globalisation.js]: Only thing I was unsure of was if the constants added above should be included in the valid locales array? I'm scared to break an existing story

File: packages/storybook/constants/globalisation.js [hunk 3]
```diff
@@ -1,4 +1,10 @@
+export const AUSTRALIAN_LOCALE = 'en-AU';
```
Comment (ashleynolan) [inline packages/storybook/constants/globalisation.js]: Could we export this as an object of locales? Seems a little tidier:

```suggestion
export const LOCALES = {
    au: 'en-AU',
    uk: 'en_GB',
    …
};
```

Comment (charliearlie) [reply packages/storybook/constants/globalisation.js]: Done.

File: packages/components/organisms/f-checkout/CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,16 @@
...
+
+
+v0.68.1
+------------------------------
+*March 9, 2021*
+
+### Changed
+- Locale usage in storybook
+
+
```
Comment (AlixRichards) [inline packages/components/organisms/f-checkout/CHANGELOG.md:15]: Is it only f-checkout and f-searchbox that are using Locale in storybook or are there any other components that might need updating?

Comment (charliearlie) [reply packages/components/organisms/f-checkout/CHANGELOG.md:15]: Fortunately these are the only usages.

**Review by AlixRichards — APPROVED** [review]

**Review by FayeCarter — APPROVED** [review]

**Pull request merged**